app/presenters/document/record_presenter.rb

```diff
@@ -50,13 +51,15 @@ def creators_info
 
```

Comment:
I worry that there is too much duplication in functionality / overlap in responsibility between this method and `Document::FieldGroup::Section::ValuePresenter#display`, which for `field_group('creators')` on record `/92034/GVNRC_NMI01_13925` would return:
```ruby
{
  :title => "translation missing: en.site.object.meta-label.creators",
  :sections => [
    {
      :title => "Creator",
      :items => [
        {
          :text => "Alphons Diepenbrock",
          :url => "/portal/en/explore/people/157024-alphons-diepenbrock.html",
          :ga_data => nil,
          :extra_info => nil,
          :raw => false
        }
      ],
      :is_desc => false
    }
  ],
  :id => "creators"
}
```
... which already contains most of what `#creators_info` generates.

I would suggest adding to the return value of `Document::FieldGroup::Section::ValuePresenter#display`:
```ruby
# ...
json_url: json_url,
entity: for_entity?,
europeana_entity: for_europeana_entity?
# ...
```
... defining the new methods accordingly.

The template would then of course need to be adapted to handle this structure instead.

If possible, it would be prefeable to eliminate the need for this `creators_info` method, as we had many field group specific methods in the past, but refactored into the YAML config file for field groups with reusable display logic.